SPECS/xapi.spec

```diff
@@ -466,6 +469,9 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Thu Oct 31 2024 Thierry Escande <thierry.escande@vates.tech> - 1.249.38-1.2
```

Comment:
Since upstreaming the patch first took some time, I think we can update the date here.

Reply:
Did it but the change is still in my staging area :/ Sorry for that...